packages/core/components/RichTextEditor/components/BlockStyleSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEditorState, Editor as EditorType } from "@tiptap/react";
+import { HeadingLevel, RichTextConfigType } from "../../../types";
+import styles from "../styles.module.css";
+import getClassNameFactory from "../../../lib/get-class-name-factory";
+const getClassName = getClassNameFactory("Input", styles);
+export const BlockStyleSelect = ({
```

Comment:
Nit: can we break these out with new lines between imports, getClassName and the component for readability?

Reply:
I've fixed the formatting here now.